ixwebsocket/IXWebSocket.h

```diff
@@ -113,6 +113,11 @@ namespace ix
         WebSocketSendInfo ping(const std::string& text);
         void close();
 
+        /**
```

Comment:
I'm using // more tidy comment for now, not the doxygen style. I'd like to stick to the current function style.